### PR TITLE
Bump version and get iris 3.0.0rc0 from conda-forge/rc_iris.

### PR DESCRIPTION
**Pull request by pp-mo**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by pp-mo** [comment]
@conda-forge-admin, please rerender

**Comment by pp-mo** [comment]
@conda-forge-admin, please rerender

**Comment by pp-mo** [comment]
@conda-forge-admin, please rerender

**Comment by pp-mo** [comment]
@conda-forge-admin, please rerender

**Comment by pp-mo** [comment]
@conda-forge-admin, please rerender

**Comment by pp-mo** [comment]
@conda-forge-admin, please rerender

**Comment by pp-mo** [comment]
@conda-forge-admin, please rerender

**Comment by pp-mo** [comment]
hi @ocefpaf I wonder if you can suggest anything here ?
I've been trying various things, to get the iris-grib to fetch the Iris 3.0.0rc pre-release from the labelled pre-release channel "conda-forge/label/rc_iris",
but I can't work out how or where to configure that.
Note :  I assume this is really only relevant to the _test_ phase, not the actual building

**Comment by pp-mo** [comment]
hi @ocefpaf  and some additional (!) :
 
(1) all the logs seem to contain the warning "No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.11"
    * e.g. https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=217028&view=logs&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=841356e0-85bb-57d8-dbbc-852e683d1642&l=703
Is there anything we can/should do about that ?

(2) when building locally, I get the warning "Testing based on recipes is deprecated as of conda-build 3.16.0.  Please adjust your code to pass your desired conda package to test instead."
though I don't see this in the feedstock build logs.
Again, I don't know how to fix this, there seems nothing relevant in the conda-build documentation.

TIA ! :place_of_worship:

**Comment by pp-mo** [comment]
@conda-forge-admin, please rerender

**Comment by pp-mo** [comment]
@conda-forge-admin, please rerender

**Comment by ocefpaf** [comment]
> hi @ocefpaf I wonder if you can suggest anything here ?
> I've been trying various things, to get the iris-grib to fetch the Iris 3.0.0rc pre-release from the labelled pre-release channel "conda-forge/label/rc_iris",
> but I can't work out how or where to configure that.
> Note : I assume this is really only relevant to the _test_ phase, not the actual building

I'll take a look at this in a moment. I don't think we can use conda-forge.yml in that fashion though. Let me check.

> (1) all the logs seem to contain the warning "No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.11"
    * e.g. https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=217028&view=logs&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=841356e0-85bb-57d8-dbbc-852e683d1642&l=703
Is there anything we can/should do about that ?

That is harmless. Safe to ignore.

> (2) when building locally, I get the warning "Testing based on recipes is deprecated as of conda-build 3.16.0.  Please adjust your code to pass your desired conda package to test instead."
though I don't see this in the feedstock build logs.
Again, I don't know how to fix this, there seems nothing relevant in the conda-build documentation.

What version of conda-build are you using?

**Comment by pp-mo** [comment]
@ocefpaf thanks for your time !

> What version of conda-build are you using?

It says "conda-build 3.20.3"

**Comment by pp-mo** [comment]
Stop press : After much reading, soul-searching and blatant trial+error, I think I have finally found the correct incantation for this ...
See https://github.com/conda-forge/iris-grib-feedstock/blob/10baa0aee9dfd5f094470767a1fe01e5bdceca5c/recipe/conda_build_config.yaml#L2
I will try adopting that approach here tomorrow

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,7 +19,8 @@ requirements:
     - pip
   run:
     - python
-    - iris >=2
+    - numpy >=0.15
+    - iris >=3.0.0rc0
```
Comment (bjlittle) [inline recipe/meta.yaml:23]: @pp-mo Shouldn't this just be completely opinionated i.e., `iris 3.0.0rc0` i.e., this RC version of `iris-grib` only makes sense to work with the RC version of `iris`.

Comment (pp-mo) [reply recipe/meta.yaml:23]: ? I don't agree.
When Iris 3.0 is out, it should work fine with that.

**Comment by pp-mo** [comment]
> I think I have finally found the correct incantation 

Actually, the exact info I needed was really in [cfep#05](https://github.com/conda-forge/cfep/blob/master/cfep-05.md#specification), but I missed it.

That is, in essence to control alternate channels in `recipe/conda_build_config.yaml`, rather than `conda_forge.yml`.
The advice says to specify the 'target channel label' that way, too.  I guess the advice on this in the conda-forge docs may be out of date ?
We shall see if this works in practice..

**Comment by ocefpaf** [comment]
> Stop press : After much reading, soul-searching and blatant trial+error, I think I have finally found the correct incantation for this ...

Sorry. I just found out that our docs are outdated on that. I'm sending a PR to update them.

**Comment by pp-mo** [comment]
Apologies @ocefpaf I was intending to close this + open a cleaner replacement PR.
But I am just waiting for communications with @bjlittle , while he is otherwise busy, before proceeding with that.

I agree that the extra lines in `conda_forge.yml` are not wanted.
My latest in [this branch](https://github.com/pp-mo/iris-grib-feedstock/tree/rc_0v16) , shortly to become the new PR.

**Comment by pp-mo** [comment]
As this has got messy, I'm closing it in favour of a [replacement PR](https://github.com/conda-forge/iris-grib-feedstock/pull/25).

**Comment by ocefpaf** [comment]
No problem. I'm slow this week. Annual day job meeting and Python 3.9 migration on conda-forge. But I guess you know what to do now. (I'm updating the docs on that BTW.)

**Pull request closed without merging**